Libraries/acdriver/CMakeLists.txt

```diff
@@ -14,11 +14,17 @@ add_library(acdriver SHARED
             Sources/Result.cpp
             Sources/VersionAction.cpp
             Sources/CompileAction.cpp
+            Sources/CompileActionImageSet.cpp
+            Sources/CompileOutput.cpp
```

Comment:
Yeah, definitely. The tests for `acdriver` tests `Result`, `Options`, and `Output` already.